src/util.js

```diff
@@ -8,21 +8,113 @@ export function getMediaStreamConstraints(facingMode, frameRate) {
   }
 }
 
-export function getConfidentPoses(poses, minPoseConfidence, minPartConfidence) {
-  return poses
-    .filter(({ score }) => score > minPoseConfidence)
-    .map(pose => ({
-      ...pose,
-      keypoints: pose.keypoints.filter(({ score }) => score > minPartConfidence)
-    }))
+export function getConfidentPoses(poses, minPoseConfidence) {
+  return poses.filter(({ score }) => score > minPoseConfidence)
 }
 
+// Draws lines between points of the body to form a wire-frame of sorts to better understand the poses
+export function drawSegment([ay, ax], [by, bx], color, scale, ctx) {
+  ctx.beginPath()
+  ctx.moveTo(ax * scale, ay * scale)
+  ctx.lineTo(bx * scale, by * scale)
+  ctx.lineWidth = 3
+  ctx.strokeStyle = color
+  ctx.stroke()
+}
+
+const MIN_CONFIDENCE = 0.2
+
 export function drawKeypoints(ctx, keypoints) {
```

Comment:
My intent was drawKeypoints accept min confident keypoints which already filtered

so How about change drawKeypoints like this?
```js
export function drawKeypoints(ctx, keypoints)
```
to
```js
export function drawKeypoints(ctx, minConfidentKeypoints)
```

then  you don't need to filter min part confident again in this function